metals/src/main/scala/scala/meta/internal/metals/SemanticDBIndexer.scala

```diff
@@ -0,0 +1,66 @@
+package scala.meta.internal.metals
+
+import java.nio.file.Path
+import MetalsEnrichments._
+import java.nio.file.Files
+import ch.epfl.scala.bsp4j.ScalacOptionsResult
+import scala.meta.internal.semanticdb.TextDocuments
+
+class SemanticDBIndexer(
```

Comment:
nit:
```suggestion
class SemanticdbIndexer(
```

In written documentation the correct capitalization is `SemanticDB` but for better or worse, we use `SemanticdbFoo` in code since I didn't like how `SemanticDBFoo` reads

Reply:
Changed